### ISSUE TITLE
add additional unit test for cpld_gridgen for vertex retrievals

### ISSUE THREAD

**Issue by DeniseWorthen** [NOAA-EMC/UFS_UTILS#649]
A second unit test should be added for the cpld_gridgen utility which tests the retrieval of the vertex points at each stagger location.

**Comment by GeorgeGayno-NOAA** [comment]
@DeniseWorthen there have been no updates to this issue. Do you want to keep it open?

**Comment by DeniseWorthen** [comment]
Yes, I actually think about it from time to time, but I see no prospect of having time to work on it.
